CONTRIBUTING.md

```diff
@@ -44,24 +46,68 @@ go test -timeout=30s -p 1 ./...
 
-## VSCode
-
-VSCode provides first-class support for devcontainers. See:
-
-* https://code.visualstudio.com/docs/remote/remote-overview
-* https://code.visualstudio.com/docs/remote/containers
+* Clone the pmm-admin [repo](https://github.com/percona/pmm-admin/) and install it by running `make install`.
+* Clone the pmm-agent [repo](https://github.com/percona/pmm-agent).
+* Run database instances to be monitored, you can either run your own or use the [`docker-compose.yml`](https://github.com/percona/pmm-agent/blob/master/docker-compose.yml) file provided by pmm-agent to run MySQL, PostgreSQL and MongoDB containers using `make env-up` in the pmm-agent repo (make sure to comment out the `pmm-server` service in the docker-compose file since we are already running pmm-managed in devcontainer).
+* Open another shell session and `cd` into the pmm-agent repo, run `make setup-dev` and `make run` to setup and run pmm-agent and connect it to pmm-managed
+* In another shell use pmm-admin to add agents to the database instances and start monitoring them using `pmm-admin add mysql --username=root --password=root-password`, `pmm-admin add postgresql --username=pmm-agent --password=pmm-agent-password` and `pmm-admin add mongodb --username=root --password=root-password`.
+* Now pmm-managed has started monitoring the databases, login to the web client in your browser to verify. The number of monitored instances will have increased.
 
+## Working with STT
+* Add the following environment variables to `docker-compose.yml` to dowload checks from Percona's checks service
+```
+PERCONA_TEST_SAAS_HOST=check-dev.percona.com:443
+PERCONA_TEST_CHECKS_PUBLIC_KEY=RWTg+ZmCCjt7O8eWeAmTLAqW+1ozUbpRSKSwNTmO+exlS5KEIPYWuYdX
+PERCONA_TEST_CHECKS_INTERVAL=10s # set it to any low duration to execute checks quickly
+```
+* Setup the devcontainer using `make env-up` and run your changes inside it by running `make env` and then`make run`.
+* Follow the steps in the previous section to set up instances for monitoring.
+* Go to the PMM dashboard and enable STT in `PMM Settings -> Advanced Settings`.
+* Any failed checks will produce result on the dashboard.
+
+## Working with Integrated Alerting
+* Make sure you have instances setup up for monitoring.
+* First we need to setup global communication settings for notification channels. Go to "Communications" tab under "PMM Settings" and set either the SMTP config for email alerts or the Slack webhook URL for slack alerts or both.
+* Go to the "Integrated Alerting" dashboard and select the "Notification Channels" tab. Create a notification channel to send the alerts to.
+* Now create an alert rule using the built-in alert rule templates and the notification channel.
+* If the alert rule condition satisfies then the alert will be visible on the "Alerts" tab and notification sent to the channel provided that it was configured correctly.
```

Comment:
All that should be in the user documentation, and there should be just a link with a short explanation. Again, don't copy/duplicate text / instructions

Reply:
@AlekSi, suggestion is good but take the couple extra seconds to point out the link if possible: https://www.percona.com/doc/percona-monitoring-and-management/2.x/using/alerting.html 
I'd assume that if they weren't here in the first place it's because the author didn't know they existed

Reply:
Neither did I :) But I was sure it exists

Reply:
updated